TypeSystem/flow-analysis/reachability_for_A04_t02.dart

```diff
@@ -15,11 +15,50 @@
 /// Test that if `S` contains a reachable `break` then `after(N)` is reachable.
 /// @author sgrekhov22@gmail.com
 
-main() {
+class C {
+  int v;
+  C(this.v);
+}
+
+test1() {
   int i;
   for (;;) {
     break;
   }
   i = 42;
   i; // Definitely assigned.
```

Comment:
Same issue here: I think a "missing return" error may be the best way to test that code after the `for` statement is reachable, and then we probably don't need `test2-3-4`.